README.md

```diff
@@ -2,15 +2,379 @@
 
 ## Overview
 
-We're going to review the file structure we'll be using for our MVC Sinatra applications. We could, if we wanted to, write our entire app in a single file. As you might imagine, this would make things very difficult to read and debug.
+We'll review the file structure we'll be using for our MVC Sinatra applications.
+
+## Objectives
+
+1. Explain the files and folders in a Sinatra MVC file structure
+2. Describe the different folders in the `app` directory and create new files and add code to these folders
+3. Complete and run a Sinatra MVC application
+
+## Keeping Code Organized 
+
+We could, if we wanted to, write our entire app in a single file. As you might imagine, this would make things very difficult to read and debug.
+
+Keeping our code organized is crucial when developing complex applications. This concept is called `separation of concerns` and `single responsibility`. Each file in our application will have a different responsibility and we'll keep these responsibilities split up into reasonable chunks.
+
+You'll be coding along in this lesson so fork and clone this lab. There are tests to run to make sure your solutions are working. Find the instructions underneath the descriptions for each file you'll be editing.
+
+## What does a Sinatra MVC File Structure Look Like?
+
+Take a look at the file structure in this directory. It's okay if it feels overwhelming at first. We're going to walk through the different files and folders and discuss what their responsibilities are.
+
+```bash
+├── Gemfile
+├── README.md
+├── app
+│   ├── controllers
+│   │   └── application_controller.rb
+│   ├── models
+│   │   └── model.rb
+│   └── views
+│       └── index.erb
+├── config
+│   └── environment.rb
+├── config.ru
+├── public
+│   └── stylesheets
+└── spec
+    ├── controllers
+    ├── features
+    ├── models
+    └── spec_helper.rb
+```
+
+### `Gemfile`
+
+This holds a list of all the gems needed to run the application. The bundler gem provides us access to a terminal command: `bundle install`. Bundler will look in the Gemfile and install any gems, as well as any gem dependencies for this application.
+
+Go ahead and enter this command in terminal. It will create a `Gemfile.lock` file for you, which is just a documentation of what versions of the gem you have installed and should use. The lock word is actually because it makes sure that only one thing is running bundle install at a time.
```

Comment:
This isn't quite right; `Gemfile.lock` _locks down_ the versions of gem dependencies more strictly than a Gemfile alone.

From the docs:

> This is important: the Gemfile.lock makes your application a single package of both your own code and the third-party code it ran the last time you know for sure that everything worked.

http://bundler.io/v1.3/rationale.html